lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -1877,6 +1882,50 @@ def test_remove_cell_measure(self):
                          [[self.b_cell_measure, (0, 1)]])
 
 
+class Test_share_data(tests.IrisTest):
+    def setter_lazy_data(self):
+        cube = Cube(biggus.NumpyArrayAdapter(np.arange(6).reshape(2, 3)))
```

Comment:
this is also a mistake, that I missed due to the earlier mistake, of the test not running